appveyor.yml

```diff
@@ -31,14 +31,12 @@ init:
   - ps: "[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12"
 
 install:
-#  - appveyor DownloadFile https://cygwin.com/setup-x86_64.exe -FileName C:\projects\cygwin-setup-x86_64.exe
-#  - C:\projects\cygwin-setup-x86_64.exe -q -P make,p7zip,mingw64-i686-gcc-core,mingw64-x86_64-gcc-core
+  - appveyor DownloadFile https://cygwin.com/setup-x86_64.exe -FileName C:\projects\cygwin-setup-x86_64.exe
+  - C:\projects\cygwin-setup-x86_64.exe -q -D -P make,p7zip,mingw64-i686-gcc-core,mingw64-x86_64-gcc-core
+  - set PATH=c:\cygwin64\bin;c:\cygwin64\lib\p7zip
```

Comment:
Is that documented anywhere: there are plenty of sites out there that suggest that this is okay. e.g. https://cygwin.com/ml/cygwin/2006-01/msg01476.html